mautic/mautic
# TPROD-363: Deleting unused php templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

These bundles were already refactored from PHP templates to Twig but some PHP templates were forgotten or the use of them. This PR cleans it up.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. This affects many bundles. Some changes touched these features:
3. Form submit asset download action - The message should be formatted based on the theme used in the instance or form
4. SMS send action should be displayed in the contact detail's timeline
